docs/shared/sapi_rest.rst

```diff
@@ -91,11 +91,36 @@ The serialized file may look like this:
     ============== ==============================
     avg_load       Average current load for the solver.
     description    Description of the solver.
-    id             Unique ID (name) of the solver.
+    id             Unique ID (name) of the solver. Only available for an
+                   ``Accept`` request header  with ``version=2.0.0``, which is
+                   the default.

```

Comment:
Representation version varies per resource. Some are 2.0, some 2.1, etc, but they are all currently 2.x. In the future they'll be 3.x.

Also, we should refer to "resource representation version", not the accept header:

```suggestion
    id             Unique ID (name) of the solver. Only available in
                   solver representation versions 1 and 2.
```